include/utility/json_utility.hpp

```diff
@@ -638,5 +638,59 @@ inline bool isBackupAndRestoreRequired(const nlohmann::json& i_sysCfgJsonObj)
     }
     return false;
 }
+
+/**
```

Comment:
Commit message does not completely explain the functionality. It needs to mentions all the restriction and also briefly explain how it works.

Reply:
done